crates/bevy_pbr/src/render_graph/forward_pipeline/forward.frag

```diff
@@ -1,21 +1,59 @@
+// From the Filament design doc
```

Comment:
I'm thinking we should rename `forward.frag`, `forward.vert`, `ForwardPipeline`, etc to `pbr.frag`, `pbr.vert`, `PbrPipeline`. This "is" a forward render, but I think "pbr" is more specific/relevant for these shaders (and thats my bad :smile: )

Reply:
Yes, I don't think we intend to ever add a deferred renderer to Bevy officially anyway (the intention is to transition to clustered forward rendering), so "forward" as a name is a little meaningless.

Reply:
Since it's part of the `pbr` crate, I went with just `pipeline` instead of `forward_pipeline` and `pbr.(vert|frag)`. What do you think?

Reply:
Yup that looks good to me.